docs/src/api/params.md

```diff
@@ -480,10 +480,17 @@ is considered matching if all specified properties match.
 A glob pattern, regex pattern or predicate receiving [URL] to match while waiting for the navigation.
 
 ## wait-for-event-event
+* langs: js, java, python
 - `event` <[string]>
 
 Event name, same one typically passed into `*.on(event)`.
 
+## csharp-wait-for-event-event
+* langs: csharp
+- `playwrightEvent` <[PlaywrightEvent<T>]>
+
+Event type, same one typically passed into `WaitForEventAsync`.
```

Comment:
This is the main method we'll show in the snippets? I think it should not redirect to some other method for the parameter description.